### DISCUSSION TITLE
Vecchio repo

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/opendatasicilia.it#5] [Generale]
Buongiorno, avevamo creato anche il repo [opendatasicilia/sito](https://github.com/opendatasicilia/sito) in cui erano state aperte alcune [issues](https://github.com/opendatasicilia/sito/issues). Come ci muoviamo? Cestiniamo il vecchio? C'è modo di fonderli?

**Comment by gbvitrano** [comment]
Trasferisci le issue e poi se vuoi lo elimini :-)

![image](https://user-images.githubusercontent.com/20488693/143679245-d2011cdd-212b-4de9-af35-8ae7b3bf0598.png)

l'operazione va fatta per ogni singola issue

**Reply by dennisangemi** [reply]
Fatto! Grazie mille. Ho trasferito le issue. Non credo di avere le autorizzazioni per eliminare il repo (o forse non so farlo), lascio a voi ;)

**Reply by aborruso** [reply]
elimino io il repo

**Reply by dennisangemi** [reply]
Grazie mille
